packages/core/src/__tests__/snapshots.test.ts

```diff
@@ -0,0 +1,90 @@
+import { parseJsx } from '../parsers/jsx';
+import { builderContentToJsxLiteComponent } from '../parsers/builder';
+import { componentToBuilder } from '../generators/builder';
+import { componentToJsxLite } from '../generators/jsx-lite';
+import { componentToHtml } from '../generators/html';
+import { componentToLiquid } from '../generators/liquid';
+import { componentToReact } from '../generators/react';
+import { componentToSolid } from '../generators/solid';
+import { componentToVue } from '../generators/vue';
+import { TestDataLoader, TestData } from './loaders/test-data-loader';
+
+const path = require('path');
+
+const dataLoader = new TestDataLoader({ subdirectories: true });
+const testCases = dataLoader.load(path.resolve(__dirname, './data'));
+
+describe('Builder', () => {
+  testCases.forEach((data: TestData) => {
+    test(data.name, () => {
+      const jsx: string = require(data.file);
+      const json = parseJsx(jsx);
+      const builderJson = componentToBuilder(json);
+      expect(builderJson).toMatchSnapshot();
+
+      const backToJsxLite = builderContentToJsxLiteComponent(builderJson);
+      const jsxLite = componentToJsxLite(backToJsxLite);
+      expect(jsxLite).toMatchSnapshot();
+    });
+  });
+});
+
+describe('Html', () => {
+  testCases.forEach((data: TestData) => {
+    test(data.name, () => {
+      const jsx: string = require(data.file);
```

Comment:
I don't think we need to use `require` and invoke Node's module resolution algorithm, using `fs.readFileSync` should be enough, something like:
```suggestion
      const jsx = fs.readFileSync(data.file, 'utf8');
```